### PR TITLE
Added project service

### PR DESCRIPTION
**Pull request by ywkim312**
Added project services

### CODE REVIEW

File: templates/svc-project/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "incore.fullname" . }}-services
+                  key: COMMON_MONGODB_URI
+            - name: SERVICES_ALLOW_ORIGIN
+              value: {{ join "," .Values.cors.origin | quote }}
+            - name: SERVICES_ALLOW_METHODS
+              value: {{ .Values.cors.methods | quote }}
+            - name: SERVICES_ALLOW_HEADERS
+              value: {{ .Values.cors.headers | quote }}
```
Comment (longshuicy) [inline templates/svc-project/deployment.yaml:50]: Is it possible to add data/dfr3/hazard service URI here. In the future I might need to make internal calls to those services from project service. 
e.g.

```
 - name: SERVICES_URL
              valueFrom:
                secretKeyRef:
                  name: {{ include "incore.fullname" . }}-services
                  key: SERVICES_URL
```

Comment (longshuicy) [reply templates/svc-project/deployment.yaml:50]: Actually i'll create a PR against this.

**Review by longshuicy — APPROVED** [review]

**Review by Rashmil-1999 — APPROVED** [review]

File: values.yaml [hunk 2]
```diff
@@ -351,6 +351,20 @@ svc_sema:
 #            average: 10
 #            burst: 20
 
+# ----------------------------------------------------------------------
+# PROJECT SERVICE CONFIGURATION
+# ----------------------------------------------------------------------
+svc_project:
+  replicas: 4
```
Comment (robkooper) [inline values.yaml:358]: can we have the default be 1, and change this on prod to be 4

Comment (ywkim312) [reply values.yaml:358]: Chen is creating a new PR now. We made all the replica counts as 1 in there. I will update the value-incore-prod.yaml now

**Pull request merged**